src/lib/components/Frontpage.svelte

```diff
@@ -0,0 +1,61 @@
+<script>
+	import { Button, Card, Gallery } from 'flowbite-svelte';
+	import { ArrowRightOutline } from 'flowbite-svelte-icons';
+
+	// Define the styles for the header, paragraph, arrow, and button
+	const headerStyle = 'mb-2 text-2xl font-bold tracking-tight text-gray-900 dark:text-white';
+	const paragraphStyle = 'mb-3 font-normal text-gray-700 dark:text-gray-400 leading-tight';
+	const arrowStyle = 'ms-2 h-6 w-6 text-white';
+	const buttonStyle = 'w-fit';
+	const galleryStyle = 'grid-cols-2 gap-y-12';
```

Comment:
2 cols looks cramped on a small screen, maybe have 1 column by default, with 2 cols for medium+ size displays?

`'grid-cols-1 md:grid-cols-2'`